src/ucp/core/ucp_ep.inl

```diff
@@ -44,9 +44,14 @@ static inline ucp_lane_index_t ucp_ep_get_tag_lane(ucp_ep_h ep)
     return ucp_ep_config(ep)->key.tag_lane;
 }
 
-static inline int ucp_ep_is_rndv_lane_present(ucp_ep_h ep)
+static inline int ucp_ep_is_rndv_lane_present(ucp_ep_h ep, ucp_lane_index_t idx)
 {
-    return ucp_ep_config(ep)->key.rndv_lane != UCP_NULL_LANE;
+    return ucp_ep_config(ep)->key.rndv_lanes[idx] != UCP_NULL_LANE;
+}
+
+static inline int ucp_ep_is_rndv_mrail_present(ucp_ep_h ep)
+{
+    return ucp_ep_config(ep)->key.rndv_lanes[1] != UCP_NULL_LANE;
```

Comment:
minor: I suggest to add an enum for the lanes - so that 1 wouldn't be hardcoded.

Reply:
here is evaluation for more than 1 lane configured. Number here is real the most clear value

Reply:
why not checking key.num_rndv_lanes?
also i would avoid using the term "multi-rail" in UCP code, and stick to UCP terminology. for example, rename this to `ucp_ep_is_rndv_single_lane()`

Reply:
fixed